### PR TITLE
Add fee schedule for Fuji

### PR DESCRIPTION
**Pull request by laviniat1996**

### CODE REVIEW

**Review by yulin-dong — CHANGES_REQUESTED** [review]
I would suggest to list out Fuji differences using the same format as that in https://docs.avax.network/nodes/validate/how-to-stake#fuji-testnet.

**Review by dubrado — APPROVED** [review]

File: docs/reference/standards/guides/txn-fees.md [hunk 1]
```diff
@@ -21,43 +21,47 @@ Different types of transactions require payment of a different transaction fee.
...
+| X        | Import AVAX               |                          0.001 |
++----------+---------------------------+--------------------------------+
+| X        | Export AVAX               |                          0.001 |
++----------+---------------------------+--------------------------------+
+| C        | Simple send               |                   >= 0.001575* |
++----------+---------------------------+--------------------------------+
+
+On Fuji Testnet, all fees are the same as those on Mainnet, except the following ones:
+- Create Subnet: 0.1 AVAX
+- Create Blockchain: 0.1 AVAX
```
[the remark this answers is not in the document]
Comment (laviniat1996) [reply docs/reference/standards/guides/txn-fees.md:64]: @yulin-dong not sure what's wrong with that info?

[the remark this answers is not in the document]
Comment (laviniat1996) [reply docs/reference/standards/guides/txn-fees.md]: @yulin-dong fixed b521798d621e6928a32a42d08d52619db9ce5d87

[the remark this answers is not in the document]
Comment (yulin-dong) [reply docs/reference/standards/guides/txn-fees.md]: @laviniat1996 please fix build

[the remark this answers is not in the document]
Comment (laviniat1996) [reply docs/reference/standards/guides/txn-fees.md]: @yulin-dong done

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**